Neos.Flow/Classes/Persistence/Doctrine/EntityManagerFactory.php

```diff
@@ -141,4 +150,30 @@ public function emitBeforeDoctrineEntityManagerCreation(Connection $connection,
     public function emitAfterDoctrineEntityManagerCreation(Configuration $config, EntityManager $entityManager)
     {
     }
+
+    /**
+     * The logger middleware needs to be set on the configuration applied to the Driver and Connection
+     *
+     * @param class-string $configuredSqlLogger
+     * @param Configuration $doctrineConfiguration
+     * @return Configuration
+     * @throws InvalidConfigurationException
+     */
+    protected function enableSqlLogger(string $configuredSqlLogger, Configuration $doctrineConfiguration): Configuration
+    {
+        $sqlLoggerInstance = $this->objectManager->get($configuredSqlLogger);
+        if (!$sqlLoggerInstance instanceof SQLLogger) {
+            throw new InvalidConfigurationException(sprintf('Neos.Flow.persistence.doctrine.sqlLogger must be \Neos\Flow\Persistence\Doctrine\Logging\SqlLogger, %s given.', get_class($sqlLoggerInstance)), 1426150388);
```

Comment:
I see a mismatch between the name checked (`SQLLogger`) and the name in the exception message (`…\SqlLogger`), and think that should be fixed.
```suggestion
        if (!$sqlLoggerInstance instanceof SqlLogger) {
            throw new InvalidConfigurationException(sprintf('Neos.Flow.persistence.doctrine.sqlLogger must be \Neos\Flow\Persistence\Doctrine\Logging\SqlLogger, %s given.', get_class($sqlLoggerInstance)), 1426150388);
```